test/functional/lib/ipfs-request-workarounds.test.js

```diff
@@ -44,8 +53,14 @@ describe('modifyRequest processing', function () {
         url: `https://ipfs.io/ipfs/${cid}`,
         initiator: 'https://some-website.example.com' // Chromium
       }
-      expect((await modifyRequest.onBeforeRequest(request)).redirectUrl)
-        .to.equal(`http://127.0.0.1:8080/ipfs/${cid}`)
+      ensureCallRedirected({
+        modifiedRequestCallResp: await modifyRequest.onBeforeRequest(request),
+        MV2Expectation: `http://127.0.0.1:8080/ipfs/${cid}`,
+        MV3Expectation: {
+          origin: '^https?\\:\\/\\/ipfs\\.io',
+          destination: 'http://127.0.0.1:8080'
```

Comment:
:warning:  Not blocking this PR, but please see https://github.com/ipfs/ipfs-companion/issues/1253 for follow-up work.